### PR TITLE
Double traction status added to standing grab

### LINKED ISSUE #621: Double Traction not applied to standing grabs
It seems as though double traction is not being applied to standing grabs. This causes characters to slide quite a bit after a jump cancelled grab. The issue this presents is that jump cancel grabs already afford a lot of benefits over dash grab and having it also travel a farther distance further neglects any utility dash grabs would have.

### PR DESCRIPTION
**Pull request by MadVillainLordQ**
Resolves #621

### CODE REVIEW

**Review by jobrien97 — DISMISSED** [review]
lgtm

**Comment by jobrien97** [comment]
On second thought, might want to include CATCH_PULL (tether reel in), CATCH_WAIT (grab idle) and CATCH_ATTACK (pummel) as well

File: fighters/common/src/opff/physics.rs [hunk 1]
```diff
@@ -156,7 +156,11 @@ unsafe fn extra_traction(fighter: &mut L2CFighterCommon, boma: &mut BattleObject
         *FIGHTER_STATUS_KIND_ATTACK_HI4,
         *FIGHTER_STATUS_KIND_ATTACK_LW4_START,
         *FIGHTER_STATUS_KIND_ATTACK_LW4_HOLD,
-        *FIGHTER_STATUS_KIND_ATTACK_LW4
+        *FIGHTER_STATUS_KIND_ATTACK_LW4,
+        *FIGHTER_STATUS_KIND_CATCH.
```
Comment (jobrien97) [inline fighters/common/src/opff/physics.rs]: Replace period with comma

**Review by jobrien97 — CHANGES_REQUESTED** [review]

File: fighters/common/src/opff/physics.rs (hunk 1 above)
Comment (MadVillainLordQ) [reply fighters/common/src/opff/physics.rs]: Got it. Thanks for catching that.

**Review by jobrien97 — APPROVED** [review]

**Pull request merged**